content/publications.md

```diff
@@ -4,31 +4,34 @@ date: 2022-08-12T09:46:31+02:00
-* 08/2020 [Eclipse Kuksa.val DBC Feeder Demo](https://www.youtube.com/watch?v=nTzmDDy3iwQ)
-* 12/2019 [APPSTACLE and Eclipse KUKSA Interview](https://www.youtube.com/watch?v=-dOwFY2l-SY) with Sven Erik Jeroschewski (Bosch Software Innovations) and Philip Heisig (Dortmund University of Applied Sciences and Arts) at EclipseCon Europe 2019
-* 12/2019 [Using Eclipse Kuksa for Connected Vehicle Services](https://www.youtube.com/watch?v=FcqzLn-wiB0) by Sven Erik Jeroschewski, Pratheek Rai adn Sebastian Lohmeier at EclipseCon Europe 2019
-* 05/2019 [Bridging Eclipse SUMO with Eclipse Kuksa](https://youtu.be/FuIaJ2tlnRE) by Philipp Heisig, Fachhochschule Dortmund
-* 03/2019 [Appstacle Use Case: Remote Driver Authentication](https://youtu.be/eZHSCXSc2wc). KUKSA PoC by OTOKAR and Netaş
-* 11/2018 [Automotive meets IoT: Innovating your future vehicle](https://www.youtube.com/watch?v=tD8pt7WMbuQ) by Sebastian Schildt, Bosch
-* 05/2018 [APPSTACLE interview](https://www.youtube.com/watch?v=qCyBjZnGK2E) with Robert Höttger Fachhochschule Dortmund - University of Applied Science and Art at Eclipsecon France
+
+- 04/2022 [KUKSA CANOPi Hardware for SDV development](https://www.youtube.com/watch?v=y6zAF-tSS2Q)
+- 06/2021 [Eclipse KUKSA.val for SCR Anti-Tampering Monitoring in Heavy Vehicles](https://www.youtube.com/watch?v=20U8bWwWfgw), ([Slides](https://events.eclipse.org/2021/saam-mobility/presentations/D2-02-Presentation.pdf)) at Eclipse SAAM 2021 by Sebastian Schildt, Bosch
+- 08/2020 [Eclipse Kuksa.val DBC Feeder Demo](https://www.youtube.com/watch?v=nTzmDDy3iwQ)
+- 12/2019 [APPSTACLE and Eclipse KUKSA Interview](https://www.youtube.com/watch?v=-dOwFY2l-SY) with Sven Erik Jeroschewski (Bosch Software Innovations) and Philip Heisig (Dortmund University of Applied Sciences and Arts) at EclipseCon Europe 2019
+- 12/2019 [Using Eclipse Kuksa for Connected Vehicle Services](https://www.youtube.com/watch?v=FcqzLn-wiB0) by Sven Erik Jeroschewski, Pratheek Rai adn Sebastian Lohmeier at EclipseCon Europe 2019
+- 05/2019 [Bridging Eclipse SUMO with Eclipse Kuksa](https://youtu.be/FuIaJ2tlnRE) by Philipp Heisig, Fachhochschule Dortmund
+- 03/2019 [Appstacle Use Case: Remote Driver Authentication](https://youtu.be/eZHSCXSc2wc). KUKSA PoC by OTOKAR and Netaş
+- 11/2018 [Automotive meets IoT: Innovating your future vehicle](https://www.youtube.com/watch?v=tD8pt7WMbuQ) by Sebastian Schildt, Bosch
+- 05/2018 [APPSTACLE interview](https://www.youtube.com/watch?v=qCyBjZnGK2E) with Robert Höttger Fachhochschule Dortmund - University of Applied Science and Art at Eclipsecon France
+- 03/2023 [Vehicle Signaling Specification and KUKSA.val in AGL](https://www.youtube.com/watch?v=RhSocQDu_DY) by Scott Murray, Konsulko Group at AGL All Member Meeting
```

Comment:
You added the newest stuff to the wrong side (It is at the bottom)

Reply:
Oh yes, didn't look closely enough :)